### DISCUSSION TITLE
How to load a model saved by Pycaret without the load_model() function

### DISCUSSION THREAD

**Discussion by MaximeTut** [pycaret/pycaret#3037] [Q&A]
Hi,
I have trained a pipeline model with pycaret and I want to load it to predict new data in a new file.py. But I have defined a different virtual environnement in the file I want to load it. I can not import pycaret in this virtual env because of conflicts. So I can't load the model with the function load_model()

I tried to load it with pickle library but without success..

Does anyone have an idea ?

Many thanks

**Comment by MaximeTut** [comment]
I found a solution : 
import joblib
joblib.load("file.pkl")

**Comment by Rodrigolorcas** [comment]
Nice, I also tried and found that. Would be awesome if there's a way to save the pipeline
